### PR TITLE
Render Note field tokens correctly - they are already HTML.

### PR DESCRIPTION
**Pull request by aydun**
Overview
----------------------------------------
Extracted from #13174 and #12012  Small bug fix so that tokens of Note fields are rendered correctly.  They are already HTML so don't convert them to HTML again.

### CODE REVIEW

File: Civi/Token/TokenRow.php [hunk 1]
```diff
@@ -225,6 +226,10 @@ public function fill($format = NULL) {
               if ($entity == 'activity' && $field == 'details') {
                 $htmlTokens[$entity][$field] = $value;
               }
+              elseif (\CRM_Utils_Array::value('data_type', \CRM_Utils_Array::value($field, $entityFields['values'])) == 'Memo') {
+                // Memo fields aka custom fields of type Note are html.
+                $htmlTokens[$entity][$field] = $value;
```
Comment (mattwire) [inline Civi/Token/TokenRow.php]: @seamuslee001 Could you review from a security perspective?  Are we ok to insert html directly into tokens?

Comment (colemanw) [reply Civi/Token/TokenRow.php]: @aydun this is more likely to be considered secure if you use the `CRM_Utils_String::purifyHTML() function`

Comment (aydun) [reply Civi/Token/TokenRow.php]: Thanks @colemanw - changed now

**Comment by mattwire** [comment]
test this please

**Comment by aydun** [comment]
Hmm - the test is not failing locally.

Jenkins retest please.

**Comment by aydun** [comment]
test this please

File: Civi/Token/TokenRow.php (hunk 1 above)
Comment (eileenmcnaughton) [reply Civi/Token/TokenRow.php]: @colemanw @seamuslee001 with this change is it mergeable?

**Comment by seamuslee001** [comment]
Given Aydun has responded to Coleman's comment and it looks sensible to me i'm good with merging this

**Comment by aydun** [comment]
Thanks @seamuslee001

**Pull request merged**